app/views/user_mailer/poke_inactive.html.erb

```diff
@@ -3,16 +3,22 @@ Hello <%= @user.github %>,<br /><br />
 
 You signed up to make a difference by triaging issues in github, which is great, but you didn't actually subscribe to any projects. To fix this just click on a link below, sign in, then click on the giant "subscribe" button:<br /><br />
 
-Help the repo with the most issues:
-<h3><%= link_to @most_repo.full_name, repo_url(@most_repo) %></h3><br />
+<% if @most_repo.present? %>
+  Help the repo with the most issues:
+  <h3><%= link_to @most_repo.full_name, repo_url(@most_repo) %></h3><br />
+<% end %>
 
-Help a repo in need:
-<h3><%= link_to @need_repo.full_name , repo_url(@need_repo) %></h3><br />
+<% if @need_repo.present? %>
+  Help a repo in need:
+  <h3><%= link_to @need_repo.full_name , repo_url(@need_repo) %></h3><br />
+<% end %>
 
-Help a random repo:
-<h3><%= link_to @random_repo.full_name, repo_url(@random_repo) %></h3><br />
+<% if @random_repo.present? %>
+  Help a random repo:
+  <h3><%= link_to @random_repo.full_name, repo_url(@random_repo) %></h3><br />
+<% end %>
 
-If you don't like any of those <%= link_to "add your own", new_repo_url %> repo, <%= link_to "browse existing " , root_url %> repos, or you can even subscribe to <%= link_to "CodeTriage itself", File.join(root_url, "/codetriage/codetriage") %>. The important part is that you pick something to triage, or you will get this same email next week, maybe with different repos.<br /><br />
+If yoy don't like any of these options, remember that they are selected based in your language preferences (<%= link_to 'click here to change them', user_url(@user) %>). Not what you're looking for? <%= link_to "add your own", new_repo_url %> repo, <%= link_to "browse existing " , root_url %> repos, or you can even subscribe to <%= link_to "CodeTriage itself", File.join(root_url, "/codetriage/codetriage") %>. The important part is that you pick something to triage, or you will get this same email next week, maybe with different repos.<br /><br />
```

Comment:
`yoy` => `you`

`based in your` => `based on your`

thoughts?

Reply:
Sounds cool.